rpc/src/v1/traits/net.rs

```diff
@@ -48,16 +49,16 @@ build_rpc_trait! {
         fn get_established_peers(&self) -> Result<Vec<SocketAddr>>;
 
         #[rpc(name = "net_addToWhitelist")]
-        fn add_to_whitelist(&self, IpAddr, Option<String>) -> Result<()>;
+        fn add_to_whitelist(&self, IpCidr, Option<String>) -> Result<()>;
```

Comment:
https://github.com/CodeChain-io/codechain/blob/master/spec/JSON-RPC.md
Would you update the spec?

Reply:
I created another pr#1321